react/src/hooks/useMetadatasetTypesQuery.tsx

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "react-query";
+import { basicFetch } from "./utils";
+
+/**
+ * Return JSON result of GET /api/metadatasettypes, or undefined if the fetch is unsuccessful.
```

Comment:
The query object is always returned, and its `data` field is what may be undefined if the fetch is unsuccessful. I noticed the comment is based on `useEnumsQuery` which is also out-of-date after I changed the hooks return the query object as opposed to just the result.